install_config/storage_examples/privileged_pod_storage.adoc

```diff
@@ -0,0 +1,206 @@
+= Mount Volumes on Privileged Pods
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap:
+
+== Overview
+
+The following provides a method for mounting persistent volumes to pods attached
+to the *privileged scc*.  For the purpose of this guide, GlusterFS will be the
```

Comment:
Capitalize "scc" throughout when referring to them, unless it's in a command or file or something.